src/compiler/types.rs

```diff
@@ -306,7 +311,13 @@ pub struct CreateProgramOptions<'config> {
 pub struct CharacterCodes;
 #[allow(non_upper_case_globals)]
 impl CharacterCodes {
+    pub const max_ascii_character: char = '';
```

Comment:
Since these are specified as numbers in the Typescript version, I believe that I correctly typed in the equivalent (to `0x7f`) character here - maybe it would be more obvious what's going on to use some integer-to-char conversion helper here?